website/reviews/listeners.py

```diff
@@ -28,11 +31,18 @@ def reviews_notification(self, creator, template, context, action):
 def reviews_submit_notification(self, recipients, context):
     # Avoid AppRegistryNotReady error
     from website.notifications.emails import get_user_subscriptions
-    from website import settings
+
+    PreprintProvider = apps.get_model('osf.PreprintProvider')
+    RegistrationProvider = apps.get_model('osf.RegistrationProvider')
 
     event_type = utils.find_subscription_type('global_reviews')
     if context['reviewable'].provider._id == 'osf':
```

Comment:
trust the document_type in the context as an invariant instead of explicitly checking the type of the provider?

Reply:
`document_type` could be any string, for preprints it can be `preprint_word` which is a custom word our provider picks, like "paper" or "publication" instead of "preprint".

Reply:
I don't really want to check the document_type, it's better to just check the provider type.